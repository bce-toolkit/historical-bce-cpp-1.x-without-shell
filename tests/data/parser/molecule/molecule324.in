(NH2)2CNH
