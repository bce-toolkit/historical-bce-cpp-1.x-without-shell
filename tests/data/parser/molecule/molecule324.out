C 1
H 5
N 3
